content/contributing/core-team/contents.lr

```diff
@@ -0,0 +1,86 @@
+ - This role never changes, and continues ad infinitum.
+ - This role is different to the BDFN role. 
+
+Bee-nevolent Dictator for Now (BDFN): `Russell Keith-Magee <https://twitter.com/freakboy3742>`_
+ - A take on `Benevolent Dictator for Life <https://en.wikipedia.org/wiki/Benevolent_dictator_for_life>`_, the buck stops with the BFDN. The use of "For Now" as opposed to "For Life" is reference to the Django theme of not subjecting the responsibilities of core maintainer for a person's entire natural life. Life exists outside open source, and code/life balance and general well-being is a very important thing to keep in mind. 
+
+
+Guidelines (not actual rules)
+------------------------------------
+
+As with any project with more than one person with commit rights, there are a number of general guidelines that the team should follow: 
+
+
+ * **Be a good representation of the project to the wider community**
+
+ * **Treat every enquiry and contribution to any BeeWare project with respect.**
+
+ * **Assume everyone has good intentions, even if they haven't chosen their words well.**
+
+ * **Assume that if someone has done something the "wrong" way, it's because we've failed in communicating process.**
+ 
+ * **Assume any expression of anger or frustration comes from a genuine place of wanting to use a BeeWare tool/library.**
+
+ * No Apiarist should commit their own code
+     - Exception: "Something is very broken and needs to be fixed now"
+     - Exception: BDFN (this may change in the future)
+
+ * All code submitted for review by a core team member should be reviewed by another team member
+     - Exception: BDFN (this may change in the future)
+
+ * All code should pass Continuous Integration tests before being merged
+    - Exception: code that is known to be broken and needs to be committed for other reasons
+    - Exception: code in a repo with insufficient CI tests
+    - Exception:  Working and committed is better than perfect and not.
+
+ * Acceptance processes should be automated wherever possible
+    - This means tests, linting, spell checking, coverage, and more. This is a `work in progress <http://pybee.org/project/projects/tools/beefore/>`_
+
+
+
+Becoming an Apiarist
+----------------------------
+
+Introduction of a new Apiarist into the team is at the sole discretion of the existing Core Team. While there are no solid rules to this, in general, someone will be invited to be an Apiarist on a BeeWare project if they have demonstrated solid contributions to the project. This can also be extended to someone with specific domain knowledge (for example, iOS/macOS) which might be lacking in the existing team. It also doesn't have to be based on commits. Anyone who is able to demonstrate a vested interest in the project in general may ask to be given permission to commit to the project. 
+
+All new apiarists will be 'inducted' (for lack of a better word) in the core values and guidelines of the project. 
```

Comment:
Add: A summary of the core values can be found [on the about page](http://pybee.org/project/about/). Anyone who joins the team will be expected to uphold those values, and contribute to discussions about evolving those values over time.